go.mod

```diff
@@ -3,18 +3,19 @@ module github.com/redhat-appstudio/service-provider-integration-oauth
 go 1.17
 
 require (
+	github.com/alexedwards/scs v1.4.1
```

Comment:
is this completely new dependency? Do we know it's stable and trustworthy?

Reply:
It is a new dependency. It is used by some higher profile projects like https://github.com/solo-io/gloo and itself has quite a lot of stars on github. Didn't do any code analysis on it though,